BookingsApi/BookingsApi.DAL/Commands/AddJudiciaryPersonCommand.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Threading.Tasks;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.Domain;
+
+namespace BookingsApi.DAL.Commands
+{
+    public class AddJudiciaryPersonCommand : ICommand
+    {
+        public AddJudiciaryPersonCommand(Guid externalRefId, string personalCode, string title, 
```

Comment:
all this is shared with the update command? could we have an abstract base maybe?

Reply:
Done, in a base class now